client/src/common_css/flexbox-grid.scss

```diff
@@ -1,988 +0,0 @@
-    -ms-flex-align: center;
-    align-items: center;
-  }
-
-  .bottom-sm {
-    -webkit-box-align: end;
-    -ms-flex-align: end;
-    align-items: flex-end;
-  }
-
-  .around-sm {
-    -ms-flex-pack: distribute;
-    justify-content: space-around;
-  }
-
-  .between-sm {
-    -webkit-box-pack: justify;
-    -ms-flex-pack: justify;
-    justify-content: space-between;
-  }
-
-  .first-sm {
-    -webkit-box-ordinal-group: 0;
-    -ms-flex-order: -1;
-    order: -1;
-  }
-
-  .last-sm {
-    -webkit-box-ordinal-group: 2;
-    -ms-flex-order: 1;
-    order: 1;
-  }
-}
-
-@media only screen and (min-width: $minMediumDevice) {
-
-  .fcol-md,
-  .fcol-md-1,
-  .fcol-md-2,
-  .fcol-md-3,
-  .fcol-md-4,
-  .fcol-md-5,
-  .fcol-md-6,
-  .fcol-md-7,
-  .fcol-md-8,
-  .fcol-md-9,
-  .fcol-md-10,
-  .fcol-md-11,
-  .fcol-md-12,
-  .fcol-md-offset-0,
-  .fcol-md-offset-1,
-  .fcol-md-offset-2,
-  .fcol-md-offset-3,
-  .fcol-md-offset-4,
-  .fcol-md-offset-5,
-  .fcol-md-offset-6,
-  .fcol-md-offset-7,
-  .fcol-md-offset-8,
-  .fcol-md-offset-9,
-  .fcol-md-offset-10,
-  .fcol-md-offset-11,
-  .fcol-md-offset-12 {
-    box-sizing: border-box;
-    -webkit-box-flex: 0;
-    -ms-flex: 0 0 auto;
-    flex: 0 0 auto;
-    padding-right: 0.5rem;
-    padding-left: 0.5rem;
-  }
-
-  .fcol-md {
-    -webkit-box-flex: 1;
-    -ms-flex-positive: 1;
-    flex-grow: 1;
-    -ms-flex-preferred-size: 0;
-    flex-basis: 0;
-    max-width: 100%;
-  }
-
-  .fcol-md-1 {
-    -ms-flex-preferred-size: 8.33333333%;
-    flex-basis: 8.33333333%;
-    max-width: 8.33333333%;
-  }
-
-  .fcol-md-2 {
-    -ms-flex-preferred-size: 16.66666667%;
-    flex-basis: 16.66666667%;
-    max-width: 16.66666667%;
-  }
-
-  .fcol-md-3 {
-    -ms-flex-preferred-size: 25%;
-    flex-basis: 25%;
-    max-width: 25%;
-  }
-
-  .fcol-md-4 {
-    -ms-flex-preferred-size: 33.33333333%;
-    flex-basis: 33.33333333%;
-    max-width: 33.33333333%;
-  }
-
-  .fcol-md-5 {
-    -ms-flex-preferred-size: 41.66666667%;
-    flex-basis: 41.66666667%;
-    max-width: 41.66666667%;
-  }
-
-  .fcol-md-6 {
-    -ms-flex-preferred-size: 50%;
-    flex-basis: 50%;
-    max-width: 50%;
-  }
-
-  .fcol-md-7 {
-    -ms-flex-preferred-size: 58.33333333%;
-    flex-basis: 58.33333333%;
-    max-width: 58.33333333%;
-  }
-
-  .fcol-md-8 {
-    -ms-flex-preferred-size: 66.66666667%;
-    flex-basis: 66.66666667%;
-    max-width: 66.66666667%;
-  }
-
-  .fcol-md-9 {
-    -ms-flex-preferred-size: 75%;
-    flex-basis: 75%;
-    max-width: 75%;
-  }
-
-  .fcol-md-10 {
-    -ms-flex-preferred-size: 83.33333333%;
-    flex-basis: 83.33333333%;
-    max-width: 83.33333333%;
-  }
-
-  .fcol-md-11 {
-    -ms-flex-preferred-size: 91.66666667%;
-    flex-basis: 91.66666667%;
-    max-width: 91.66666667%;
-  }
-
-  .fcol-md-12 {
-    -ms-flex-preferred-size: 100%;
-    flex-basis: 100%;
-    max-width: 100%;
-  }
-
-  .fcol-md-offset-0 {
-    margin-left: 0;
-  }
-
-  .fcol-md-offset-1 {
-    margin-left: 8.33333333%;
-  }
```

Comment:
Did a random spot check of dropped class names, found this one being used in `tags_related_to_subject_panels.js` (don't see that being updated in this PR). Please double check for any deprecated `flexbox-grid.scss` classes that you may have missed, either consider recreating them in `bootstrap-wet-fixes-extensions.scss` or refactoring the layouts that used them.

Reply:
Wait, scratch that, `tags_related_to_subject_panels.js` was using the col version, not fcol. Weird, thought we'd switched all cols to fcols during the bootstrap 4 upgrade.